src/plugins/home/server/services/sample_data/data_sets/ecommerce/saved_objects.ts

```diff
@@ -10,6 +10,7 @@
 
 import { i18n } from '@kbn/i18n';
 import { SavedObject } from '@kbn/core/server';
+import { defaultUserContentAttributes } from '@kbn/user-content-plugin/server';
```

Comment:
Instead of having the `home` plugin knowing about the `user_content` plugin and do this import we probably want to put an extension point in place where plugins can add modifiers to the sample data before it is loaded.

In this case the `user_content` plugin would register a modifier in the `home` plugin to add the `defaultUserContentAttributes`.